virtcontainers/qemu_amd64.go

```diff
@@ -129,6 +135,13 @@ func (q *qemuAmd64) cpuModel() string {
 	if q.nestedRun {
 		cpuModel += ",pmu=off"
 	}
+
+	// VMX is not migratable yet, see https://bugzilla.redhat.com/show_bug.cgi?id=1689216
+	// issue: https://github.com/kata-containers/runtime/issues/1750
```

Comment:
How about turning this comment into a structured log call? That way:

- admins can see why performance may be suffering.
- We can tweak the [stale skip checker](https://github.com/kata-containers/tests/blob/master/.ci/kata-find-stale-skips.sh) to look at *all* github Kata issue URLs in the logs (and the code too I guess) and warn / error if it finds any that are closed as that's a sign we forgot to update something ;)

I'd just reference the Kata issue as that has the bugzilla on it.

Reply:
https://github.com/kata-containers/tests/issues/1655 raised for the stale skip idea.

Reply:
ack that a log message might be nice.